README.md

```diff
@@ -3,6 +3,56 @@ React Native Tipsi custom UI elements
 
 ## Components
 
+### `<Counter />`
+
+Component to change the number by press "+" or "-".
+
+#### Counter Props
+
+| Name | Desc | Type | Default
+| --- | --- | --- | --- |
+| `step` | Step of counting.  | Number | `1`
+| `startValue` | Initial value of counter. | Number | `1`
+| `onValueChange` | Handle value changes. | Function | `() => {}`
+
+#### Counter Themes
+
+Theme structure:
+
+```js
+{
+  container: <View />,
+  item: <View />,
+  center: <View />,
+  left: <View />,
+  right: <View />,
+  expText: <Text />,
+  centerText: <Text />,
+}
+```
+
+#### Example
+
+```js
+import React from 'react'
+import { Counter } from 'tipsi-ui-kit'
+
+const handleValueChange = (id) => (console.log(`Current value is ${id}`)
+
+const Example = () => (
+  <Counter
+    step={5}
+    startValue={25}
+    onValueChange={this.handleValueChange}
+  />
+)
+```
```

Comment:
@fAns1k your example has to errors. Did you see my comment https://github.com/tipsi/tipsi-ui-kit/pull/9#discussion_r97569528 ?